index.html

```diff
@@ -4,22 +4,94 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <title>LP</title>
-  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap" rel="stylesheet">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-  <link rel="stylesheet" href="main.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="assets/css/main.scss">
 </head>
 <body>
 
-<header class="header">
-  <nav class="nav justify-content-end">
-    <a class="nav-link active" href="#">Главная</a>
-    <a class="nav-link" href="#">Наши проекты</a>
-    <a class="nav-link" href="#">Услуги</a>
-    <a class="nav-link" href="#">Новости</a>
-    <a class="nav-link" href="#">Контакты</a>
-  </nav>
-  <img src="./assets/img/cat-1.jpg" alt="">
-</header>
+<div class="container-fluid header">
+
+  <div class="row">
+    <div class="navigation col text-white">
+      <div class="background"></div>
+      <ul>
+        <li class="active">
+          <a href="#">Главная</a>
+        </li>
+        <li>
+          <a href="#">Наши проекты</a>
+        </li>
+        <li>
+          <a href="#">Услуги</a>
+        </li>
+        <li>
+          <a href="#">Новости</a>
+        </li>
+        <li>
+          <a href="#">Контакты</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="title col-6 offset-3 text-white text-center">
+      <div class="mb-3">Передовая IT студия</div>
+      <h1 class="mb-5">Мы создаем легкие решения сложных задач и проблем</h1>
+      <a class="btn btn-primary mb-3" href="#" role="button">Наши проекты</a>
+    </div>
+  </div>
+
+</div>
+
+<!--services---------------------------->
+
+<div class="container services">
+
+  <div class="row">
+    <div class="col col-8 offset-2 text-center">
+      <h2 class="mb-4">Мы создаем мобильные приложения для крупных корпораций</h2>
+      <p class="mb-5">
+        Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне.
+        Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. </p>
+    </div>
+  </div>
+
+  <div class="row mb-5">
+    <div class="item col-4 d-flex flex-row">
```

Comment:
to replace with smth like
```suggestion
    <div class="item col-4 my-flex">
    
    .my-flex {
       ...
    }
```